docs/source/api.rst

```diff
@@ -1,5 +1,84 @@
 AA Toolbox API
 ==============
 
-.. automodule:: aatoolbox
+Data sources
+------------
+
+CHIRPS
+^^^^^^
+
+.. automodule:: aatoolbox.datasources.chirps.chirps
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+Common Operational Datasets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. automodule:: aatoolbox.datasources.codab.codab
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+FEWS NET
+^^^^^^^^
+
+.. automodule:: aatoolbox.datasources.fewsnet.fewsnet
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+IRI
+^^^
+
+Seasonal forecasts
+""""""""""""""""""
+
+.. automodule:: aatoolbox.datasources.iri.iri_seasonal_forecast
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+NDVI (USGS eMODIS)
+^^^^^^^^^^^^^^^^^^
+
+Products
+""""""""
+
+.. automodule:: aatoolbox.datasources.usgs.ndvi_products
    :members:
+   :undoc-members:
+   :show-inheritance:
+
+Base class
```

Comment:
I don't think we need to include this

Reply:
Found how to do what I wanted, show the inherited methods, in another way.